index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Asap:wght@500&family=Poppins:wght@100;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- Nav Bar  -->
+
+    <header class="nav-bar">
+      <div class="header-logo">
+        <a href="#" class="header-logo-link"><h3>My Logo</h3></a>
+      </div>
+
+      <button type="button" class="nav-button">
+        <img
+          src="./styling-features-pictures/Union.png"
+          alt="nav-button-menu"
+        />
+      </button>
```

Comment:
- [ ] Kindly make your logo smaller, and you can reduce the left and right padding, to match the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/158728991-eb738935-b7d2-4bc3-bd73-c5a7f4cc2460.png) | ![image](https://user-images.githubusercontent.com/35535130/158729075-63d2b947-001a-46c5-99dd-04180b3f4f42.png) |